CriteoPublisherSdk/Tests/UnitTests/Standalone/CRBannerViewTests.m

```diff
@@ -248,9 +248,23 @@ - (void)testCreateWebViewWithConfiguration {
   [self cr_waitShortlyForExpectations:@[ openInBrowserExpectation ]];
 }
 
-#pragma inhouseSpecificTests
-
 - (void)testLoadingSuccess {
```

Comment:
Could you rename this method to be more specific on what you're testing, something like `testLoadingAdDisplayDataSuccess` would help to understand what is tested.

Reply:
Done